client/my-sites/plugins/categories/use-categories.tsx

```diff
@@ -33,24 +33,7 @@ export const ALLOWED_CATEGORIES = [
 	'paid',
 ];
 
-export function useCategories(
-	allowedCategories = ALLOWED_CATEGORIES
-): Record< string, Category > {
-	const { __ } = useI18n();
-	const siteId = useSelector( getSelectedSiteId ) as number;
-
-	const isJetpack = useSelector(
-		( state ) => isJetpackSite( state, siteId ) && ! isAtomicSite( state, siteId )
-	);
-
-	// Only showing these top level categories for now
-	const allowed = allowedCategories.slice();
-
-	// Jetpack sites shouldn't see paid plugins
-	if ( isJetpack && allowed.indexOf( 'paid' ) >= 0 ) {
-		allowed.splice( allowed.indexOf( 'paid' ), 1 );
-	}
-
+export function getCategories( allowed = ALLOWED_CATEGORIES ): Record< string, Category > {
```

Comment:
Needed to extract this into a function in order to construct the categories server prefetch query.